f2/src/theme/style.css

```diff
@@ -292,6 +295,8 @@ body {
 .custom-control-label {
   padding-left: 1.5rem;
   font-size: 1.25rem;
+  line-height: 1.5rem;
```

Comment:
This makes the text non-responsive. On smaller screens the problem is worse, I think a smaller line-height is a good start but the padding-top needs to be removed in favor of another rule.